tests/unit/test_hcloud.py

```diff
@@ -55,6 +55,10 @@ def test__get_user_agent(self, client):
         user_agent = client._get_user_agent()
         assert user_agent == "hcloud-python/0.0.0"
 
+    def test_with_poll_interval(self):
```

Comment:
this test looks obsolete now.

Reply:
No, we don't test the client with the `poll_interval` elsewhere. We need to make sure, that the `poll_interval` is set on the client.